frontend/src/pages/MetadataByMunicipality/SingleMetaDataResult.jsx

```diff
@@ -100,22 +170,22 @@ const SingleMetaDataResult = ({ metadata }) => {
           <MetaDataTypeLink to={`/dataType/${metadataTypeName}`}>
             <h2>{metadataTypeName}</h2>
           </MetaDataTypeLink>
+          <FeedbackLabel hasFeedback={hasFeedback} />
           <p>{description}</p>
           <MetaDataLink to={`/dataset/${uuid}`}>Other municipalities who offer this data</MetaDataLink>
         </MetaDataDescription>
-        <URLWrapper href={url}>
-          <URL>{url}</URL>
-          <DataFormat>{formatName}</DataFormat>
-        </URLWrapper>
+        <MetadataURL url={url} formatName={formatName} />
       </MetaDataContent>
-      {/* TODO: Uncomment for comments
       <MetaDataRating>
-        <CommentsIcon href={`URL-TO-METADATA-PAGE-${uuid}`}>
-          <img src={chatIcon} alt="Feedback" />
-          <p>[number] Comments</p>
-        </CommentsIcon>
+        <Favourite onClick={handleLike}>
+          {isLiked ? <StarFullStyled /> : <StarEmptyStyled />}
+          <p>{likes}</p>
+        </Favourite>
+        <CommentsLink to={`viewData/dataset/${uuid}`}>
```

Comment:
This url doesnt work after the splash page got updated

Reply:
This should have been resolved :thinking:

Reply:
Fixed this right now.